modules/ROOT/pages/architecture.adoc

```diff
@@ -50,10 +50,17 @@ image::architecture-network.png[]
 
 This diagram shows the TCP load balancer used to load balance requests across the internal load balancers running on the controllers. It also shows the internal load balancer that distributes requests to each replica of Mule applications running on the workers.
 
-For internal load balancers:
+When you enable an internal load balancer, specify either of the following modes:
```

Comment:
Unsure if it makes sense to write this in the architecture section, since we don't expect the user to follow a procedure in the platform from content on this page. It was meant to be purely informational to help the user understand how RTF works, and what they'll need to consider when installing.

Reply:
Good point! Didn't catch this when I moved some info. I've reworded this.